### DISCUSSION TITLE
GitHub page didn't update 4 days after commit

### DISCUSSION THREAD

**Discussion by vronizor** [community/community#24651] [Apps, API and Webhooks]
<p>I have updated my GitHub page built with jekyll as a do regularly but the changes haven’t been deployed yet despite the fact that my last commit was 4 days ago. I have done some commits since and still no update. I cleared the cache on my devices and still not updated. Any ideas what is going on? The last version deployed seems to be one that I committed from the GitHub website directly, vs the later ones were pushed using the GitHub Desktop App — might that be an issue? Why?</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>you have build errors,</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/6/6/66572b5a724eaeeb8ac4f8dfcad3aae349c186cd.png" data-download-href="/uploads/short-url/eBlsBPveGFOsP7hX9eyL1U5LMXH.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181085841-29c08823-25c9-4204-b912-840057b0cd5e.png" alt="image" data-base62-sha1="eBlsBPveGFOsP7hX9eyL1U5LMXH" width="558" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/6/6/66572b5a724eaeeb8ac4f8dfcad3aae349c186cd_2_558x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/6/6/66572b5a724eaeeb8ac4f8dfcad3aae349c186cd_2_837x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/6/6/66572b5a724eaeeb8ac4f8dfcad3aae349c186cd_2_1116x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/6/66572b5a724eaeeb8ac4f8dfcad3aae349c186cd_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1162×1040 79.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>your last successful build was 5 days ago,</p>
<p><img src="https://user-images.githubusercontent.com/47092464/181085842-4a0de923-bbb6-4e81-9c18-0b1161feeb98.png" alt="image" data-base62-sha1="lZG7hM0uPNvyDdhAaWdWvqSBSsF" width="425" height="97"></p>
<p>you want to remember that point and trace all the changes, that way you can easily detect the errors</p>
<p>go for small changes and make sure, before you proceed, that the changes are working and the each build is actually successful,</p>

**Comment by vronizor** [comment]
<p>Thank you very much, I was unaware of that part of the repo.</p>
